backend/src/rtp.rs

```diff
@@ -1653,16 +1709,17 @@ impl Endpoint {
 
         let decrypt_failed = incoming
             .decrypt_in_place(&self.decrypt.rtp.key, &self.decrypt.rtp.salt)
-            .is_none();
+            .is_err();
         if decrypt_failed {
-            debug!(
+            let value = format!(
                 "Invalid RTP: decryption failed; ssrc: {}, seqnum: {}, pt: {}, payload_range: {:?}",
                 incoming.ssrc(),
                 incoming.seqnum(),
                 incoming.payload_type(),
                 incoming.payload_range(),
             );
-            return None;
+            debug!("{}", value);
+            return Err(EndpointError::UndeterminedError(value));
```

Comment:
I don't think there is any consumer of this information through the error right now (before they were just taking `None` and moving on). I would prefer to leave the logging code the same and just pass up a generic error for this (no value).

Along those lines, maybe you meant to use `EndpointError::SrtpDecryptionError` here?